webapp/src/Service/DOMJudgeService.php

```diff
@@ -378,39 +378,48 @@ public function getUpdates(): array
         $contest = $this->getCurrentContest();
 
         $clarifications = [];
-        if ($contest) {
-            $clarifications = $this->em->createQueryBuilder()
-                ->select('clar.clarid', 'clar.body')
-                ->from(Clarification::class, 'clar')
-                ->andWhere('clar.contest = :contest')
-                ->andWhere('clar.sender is not null')
-                ->andWhere('clar.answered = 0')
-                ->setParameter('contest', $contest)
-                ->getQuery()->getResult();
-        }
+        $judgehosts = [];
+        $rejudgings = [];
+        $internal_errors = [];
+
+        if ($this->authorizationChecker->isGranted('ROLE_ADMIN') or $this->authorizationChecker->isGranted('ROLE_JURY')) {
```

Comment:
admin also has the jury role so you only have to check that. Also you can use `$this->checkRole('jury')`